projects/components/src/breadcrumbs/breadcrumbs.component.test.ts

```diff
@@ -82,11 +82,13 @@ describe('BreadcrumbsComponent', () => {
     const element = spectator.query(byText('First'));
     spectator.click(element!);
     expect(spy).toHaveBeenCalled();
+    spy.mockRestore();
```

Comment:
Why do we need to call on the last line?

Reply:
The restoreMocks configuration option is available to restore mocks automatically between tests.